src/pages/ListPage/ChartSection.jsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
+// Component
+import VoteButton from "components/Button/Button";
+import BorderButton from "components/Button/BorderButton";
+import Tab from "components/Tab/Tab";
+import ChartItem from "components/ChartItem/ChartItem";
+// Design
+import { ReactComponent as ChartIcon } from "assets/icons/chart.svg";
+import styles from "pages/List.module.scss";
+// MockData
+import femaleData from "mockData/femaleData";
+import maleData from "mockData/maleData";
+
+export default function ChartSection() {
+  const [sortedData, setSortedData] = useState(
+    femaleData.sort((a, b) => b.totalVotes - a.totalVotes),
+  );
```

Comment:
sort는 원본 배열을 변경하는 것으로 알고 있습니다
어차피 mockData이긴 하지만 이렇게 변경하는 것이 더 안전해보여요!

```suggestion
  const [sortedData, setSortedData] = useState(
    [...femaleData].sort((a, b) => b.totalVotes - a.totalVotes),
  );
```